TINY-10078, TINY-10097, TINY-10109, TINY-10110: Maintain non-end scroll positions, reduce flickering, and fix inconsistent autoscrolling for streamContent: true iframe dialog components (#8926)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

### Improved

> - On the Safari browser, content updates for `streamContent: true` iframe dialog components were throttled to 500ms intervals to reduce the visual impact of flickering when rapidly updating iframe content. #TINY-10097

Reviewer: ```suggestion
- Content updates for `streamContent: true` iframe dialog components on Safari are now throttled to 500ms intervals to reduce the visual impact of flickering when rapidly updating iframe content. #TINY-10097
```

### Fixed

> - On Safari and Firefox browsers, non-end scroll positions were not maintained when updating the content of a `streamContent: true` iframe dialog component. #TINY-10078
> - On the Safari browser, end scroll positions were not consistently maintained when updating the content of a `streamContent: true` iframe dialog component. #TINY-10109

Reviewer: Doesn't seem worth having two almost identical entries

```suggestion
- On Safari and Firefox browsers, scroll positions were not always maintained when updating the content of a `streamContent: true` iframe dialog component. #TINY-10078 #TINY-10109
```
